src/less/chip/chip.less

```diff
@@ -0,0 +1,60 @@
+@import "../variables/variables.less";
+
+.chip {
+    align-items: center;
+    background-color: var(--color-background-tertiary);
+    border-radius: @spacing-400;
+    box-sizing: border-box;
+    color: var(--color-foreground-primary);
+    display: inline-flex;
+    font-size: @font-size-14;
+    gap: 2px;
+    height: @spacing-400;
+    justify-content: center;
+    max-width: 320px;
+    min-width: 56px;
+    padding: @spacing-75 @spacing-150;
+}
+
+.chip__text {
+    margin-top: 2px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+button.chip__button {
+    background-color: transparent;
+    border: 2px solid transparent;
+    border-radius: 50px;
+    box-sizing: border-box;
+    flex-shrink: 0;
+    height: @spacing-300;
+    line-height: 0;
+    margin-right: -6px;
+    padding: 0;
+    width: @spacing-300;
+}
+
+button.chip__button:focus-visible {
+    // we don't currently have an appropriate token for this
+    // all of the current tokens using the prop are for disabled states
+    background-color: var(--color-neutral-3);
+    border-color: var(--color-state-focus-stroke);
+    outline: 0;
+}
+
+button.chip__button:hover {
+    // we don't currently have an appropriate token for this
+    // all of the current tokens using the prop are for disabled states
+    background-color: var(--color-neutral-3);
+}
+
+button.chip__button > svg {
+    height: 14px;
+    width: 14px;
```

Comment:
Any reason why we are dropping the size? Should leave defaults.

Reply:
The icon used in the specs doesn't exist. As a result, I had to use `close-16` and make it slightly smaller.

Reply:
So the icon is part of this issue https://github.com/eBay/skin/issues/2128
We should include that icon and then remove these overrides.